evap/staff/templates/staff_semester_export.html

```diff
@@ -18,11 +18,12 @@ <h3>{% trans 'Export' %} {{ semester.name }}</h3>
 
         <div class="card mb-3">
             <div class="card-body">
-                <p>{% trans 'Select the course types you want to export. Add multiple lines to create an export file with multiple sheets.' %}</p>
+                <p>{% trans 'Select the degrees and course types you want to export. An export will include all evaluations of courses that have at least one of the selected degrees and at least one of the selected course types. Add multiple lines to create an export file with multiple sheets.' %}</p>
```

Comment:
courses have only one type, so the "at least" doesn't apply there.

Reply:
right